searchform.php

```diff
@@ -1,8 +1,8 @@
 <form id="search" class="" method="get" action="<?php echo esc_url( home_url( '/' ) ); ?>"> 
   <div class="input-append">
 		<span aria-hidden="true" data-icon="&#xf002;"></span>
-		<input class="input-xlarge" type="text" name="s" value="<?php echo trim( get_search_query() ); ?>">
+		<input label="Search" class="input-xlarge" type="text" name="s" value="<?php echo trim( get_search_query() ); ?>">
```

Comment:
Label is not a valid attribute for `<input>` elements. To make this valid, you would need to add an `id` to the `<input>`, then add a separate `<label>` element with the appropriate text, with a `for` attribute referencing the ID of the input. 

Example: 
```html
<label for="searchquery">Search</label>
<input id="searchquery" class="input-xlarge" type="text" name="s" value="<?php echo trim( get_search_query() ); ?>">
```

Reply:
Removed improper labels from this in a commit.